Dockerfile

```diff
@@ -1,28 +1,30 @@
 FROM ubuntu:jammy
 
-RUN apt-get update
-RUN apt-get install git python3 pip -y
+RUN apt-get update -y && apt-get install -y \
+    git \
+    curl \
+    python3=3.10.* \
+    python3-dev=3.10.* \
+    python3-pip \
+    unzip
 
+# Install pyoorb from B612's fork, which includes a patch to handle fortran errors better
+ENV OORB_TAG=v1.2.1a1.dev2
+ENV OORB_VERSION="pyoorb-1.2.1a1.dev2+66b7753.dirty"
 
-# Build OpenOrb
-# The following are build requirements for oorb
-RUN apt-get install gfortran curl liblapack-dev python3-numpy wget -y
-RUN git clone https://github.com/oorb/oorb.git /oorb/
-WORKDIR /oorb/
-RUN ./configure gfortran opt  --with-pyoorb --with-python=python3 --with-f2py=/usr/bin/f2py3 --prefix=/usr/local
-RUN make
-RUN make ephem
-RUN cd /oorb/data && ./getBC430
-RUN cd /oorb/data && ./updateOBSCODE
-ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/oorb/python/"
-ENV PYTHONPATH="$PYTHONPATH:/oorb/python/"
-ENV OORB_DATA="/oorb/data"
-ENV OORB_CONF="/oorb/main/oorb.conf"
-RUN make install
+# Install oorb data
+RUN curl -L -o /tmp/oorb_data.zip \
+    https://github.com/B612-Asteroid-Institute/oorb/releases/download/$OORB_TAG/oorb_data.zip
+RUN unzip -d /opt/oorb_data /tmp/oorb_data.zip
+ENV OORB_DATA=/opt/oorb_data
+
+# Install pyoorb
+RUN export WHEEL_NAME="${OORB_VERSION}-cp310-cp310-manylinux_2_17_$(uname -m).manylinux2014_$(uname -m).whl" && \
```

Comment:
Is there a reason we need these dynamically assigned in the Dockerfile instead of added to the `setup.cfg` for precovery? I'm guessing this is to allow precovery to be installed in multiple environments and architectures combined with the fact that a `.whl` on Github can't be selected dynamically like it can from a package server.